drltrace/drltrace_frontend.cpp

```diff
@@ -302,11 +308,25 @@ _tmain(int argc, const TCHAR *targv[])
         tmp--;
     *(tmp+1) = '\0';
 
+    /* in case of default config option, we use drltrace's install path */
+    if (op_config_file.get_value().compare(DEFAULT_CONFIG_DIR) == 0) {
+        dr_snprintf(tmp_path, BUFFER_SIZE_ELEMENTS(tmp_path), "%s/drltrace.config",
+                    full_frontend_path /*install dir*/);
```

Comment:
We always call NULL_TERMINATE_BUFFER after snprintf, esp when the return value is not checked